app/globals.css

```
@@ -59,3 +59,35 @@
     @apply bg-background text-foreground;
   }
 }
+
+@import '@mdxeditor/editor/style.css';
```

Biome

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)